### PR TITLE
TX General Election Dates

### PR DESCRIPTION
**Pull request by askpatrickw**

### CODE REVIEW

**Comment by askpatrickw** [comment]
TX is sooooo big it takes 2 PRs. ;-)

File: states/texas/elections.toml [hunk 1]
```diff
@@ -46,8 +46,47 @@ sources = ["https://www.sos.state.tx.us/elections/voter/important-election-dates
...
+received_by = 2020-11-04T17:00:00
+sources = [
+  "https://www.sos.state.tx.us/elections/voter/important-election-dates.shtml",
+  "https://www.votetexas.gov/military-overseas-voters/index.html",
+  "https://www.sos.texas.gov/elections/voter/reqabbm.shtml",
+]
+
+[20201103.poll.early]
+in_person_by = 2020-10-30
+in_person_start = 2020-11-19
```
Comment (tannewt) [inline states/texas/elections.toml]:
```suggestion
in_person_start = 2020-10-19
```

**Review by tannewt — CHANGES_REQUESTED** [review]
One copy and paste issue. Good otherwise!

File: states/texas/elections.toml (hunk 1 above)
Comment (askpatrickw) [reply states/texas/elections.toml]: Thanks @tannewt  All done.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**